### ISSUE TITLE
slidesToScroll problem

### ISSUE THREAD

**Issue by Mazeland** [lukaszflorczak/vue-agile#133]
Hi Łukasz,

Thanks for the awesome Vue rewrite of slick slider without jQuery. We are implementing it right now into our project and it works great. However, we need the option 'slidesToScroll' but this is not working fine. 
We noticed that this option is not present in the README.md, but its functionality is available in the plugin. Yet, there is a problem with it (maybe that's the reason it's not documented): If you enable slidesToScroll then the slider has suddenly some empty slides at the end.

Is there a fix for this available?

Regards, 
Ruud

**Comment by lukaszflorczak** [comment]
@Mazeland Exactly, I prepared the prop at the beginning, but there was some problem and I left it for later. But now I added it to the to-do list of the next version.
